python/mach_bootstrap.py

```diff
@@ -244,14 +244,15 @@ def bootstrap_command_only(topdir):
     # because the module requires non-standard python packages
     _activate_virtualenv(topdir, is_firefox_checkout(topdir))
 
-    from servo.bootstrap import bootstrap
+    # We are not set up yet, so we always use the default cache directory
+    # for the initial bootstrap.
+    # TODO(mrobinson): Why not just run the bootstrap command in this case?
+    try:
+        servo.platform.get().bootstrap(
+            servo.util.get_default_cache_dir(topdir), '-f' in sys.argv)
+    except NotImplementedError as exception:
+        print(exception)
```

Comment:
Are we intentionally not returning non-zero exit code from the exception handler?

Reply:
Oh nice catch. I've fixed this.